spec/jobs/clean_up_inactive_users_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+RSpec.describe Jobs::CleanUpInactiveUsers do
+  let(:user) { Fabricate(:user) }
```

Comment:
These should be moved inside the `it 'should clean up the user' do` test since they're not being used elsewhere

Reply:
Or at least inside the `context 'when user is inactive' do` block.